backend/types/api.d.ts

```diff
@@ -89,7 +91,11 @@ declare namespace Components {
     }
     export interface HandlesResponse {
       publicKey: string;
-      handle: string;
+      handle: {
+        base_handle: string;
+        canonical_base: string;
+        suffix: number;
+      };
```

Comment:
Just confirming whether these handle-type changes are wanted; I noticed earlier that some code was out of sync with the OpenAPI spec, and things broke when I generated all the types. Perhaps a review of this warranted to make sure the spec & code are in sync & are what is desired.

Reply:
yes this all looks good!